scripts/pre-publish.mjs

```diff
@@ -0,0 +1,26 @@
+import { opendir, rm } from 'fs/promises';
+import { join } from 'path';
+import { URL } from 'url';
+
+async function* scan(path, cb) {
+	const dir = await opendir(path);
+
+	for await (const item of dir) {
+		const file = join(dir.path, item.name);
+		if (item.isFile()) {
+			if (cb(file)) yield file;
+		} else if (item.isDirectory()) {
+			yield* scan(file, cb);
+		}
+	}
+}
+
+const folder = new URL('../dist', import.meta.url);
+const regexp = /(?:\.d\.ts(?:\.map)?|\.tsbuildinfo)$/;
```

Comment:
Hey, would `(?:(?<!index)\.d\.ts(?:\.map)?|\.tsbuildinfo)$` work? It works okay for me in my testing but I'm not super familiar with negative lookbehinds.

Reply:
Thanks, looks like it works.